src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+
+class Display extends Component{
+
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        return(
+            <div>{this.props.result}</div>
+        )
+    }
+}
```

Comment:
- `result` should be string (enforce with `propTypes`) [see requirements[]](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)

- default value of `result` should be `'0'` (use `defaultProps`) [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]